tests/testthat/test-compile.R

```diff
@@ -25,6 +25,15 @@ test_that("string input compiles", {
   )
 })
 
+test_that("character vector input compiles", {
+  input <- c(".foo {\n  color: red;\n}\n", ".bar {\n  background-color: blue;\n}\n")
+  expected <- paste(input, collapse = "\n")
+  class(expected) <- c("css", "html", "character")
+  attr(expected, "html") <- TRUE
+
+  expect_equal(sass(input), expected)
```

Comment:
Also add `expect_equal(sass(as.list(input)), expected)`

Also check for a warning when a named vector is supplied.